HW2/P3/mandelbrot.pyx

```diff
@@ -5,9 +5,15 @@ import numpy
 cimport AVX
 from cython.parallel import prange
 
+# helper function to map AVX values in counts to the appropriate positions in out_counts
```

Comment:
This is a very clever and clean way of orienting everything in the out_counts, I had a lot of trouble doing this and my solution is not as clean.